public/manifest.json

```diff
@@ -1,11 +1,15 @@
 {
-  "short_name": "CSH React",
-  "name": "CSH React Boilerplate",
-  "start_url": "./index.html",
-  "dir": "auto",
+  "short_name": "React App",
+  "name": "Create React App Sample",
+  "icons": [
+    {
+      "src": "favicon.svg",
+      "sizes": "64x64 32x32 24x24 16x16",
+      "type": "image/x-icon"
+    }
+  ],
+  "start_url": ".",
   "display": "standalone",
-  "lang": "en",
-  "orientation": "portrait",
-  "theme_color": "#b0197e",
```

Comment:
Some of these might be nice defaults to keep

Reply:
I might have deleted these accidentally. Ill put it back later today.